debezium-connector-oracle/src/main/java/io/debezium/connector/oracle/logminer/LogMinerQueryResultProcessor.java

```diff
@@ -53,8 +54,8 @@
     private final OracleConnectorConfig connectorConfig;
     private final Clock clock;
     private final Logger LOGGER = LoggerFactory.getLogger(LogMinerQueryResultProcessor.class);
-    private long currentOffsetScn = 0;
-    private long currentOffsetCommitScn = 0;
+    private Scn currentOffsetScn = Scn.ZERO;
```

Comment:
Should it be initialized with null instead?

Reply:
Sure, sure, but we didn't have any other choice with a primitive long, right? So I'm not sure where/how that value actually will apply, but my question is whether null is the more reasonable default value.

Reply:
Sure, would just need to adjust the `warnStuck` method to account for the comparison needing to do null checks.

Reply:
How about renaming th `ZERO` to `EMPTY` and hadnli ti as a safe null object similar to `io.debezium.connector.sqlserver.Lsn.NULL`

Reply:
We could do that, but does setting the class variable to `Scn.EMPTY` vs `SCN.ZERO` any different?  I am under the impression that Gunnar wanted the variable initialized as an implicit `null`, but perhaps I misunderstood?

Reply:
It depends on where this value is used eventually; initially, I thought it may be exposed in offsets or similar, in which case I'd have preferred an actual absent value (i.e. null) over some pseudo-absent value like 0. But it seems this only actually impacts `warnStuckScn()`, in which case an actual `null` is as good as `Scn.EMPTY` (and I'd prefer that or `NULL`over `ZERO` indeed). Both should be fine. Does that help to move forward?

Reply:
It does, I'll align it as @jpechane suggested as a final touch-up.

Reply:
Alright @gunnarmorling @jpechane, hopefully everything is good to go now.  Give this a last look see and if we want to do any squashing of these commits, let me know.